geometry_msgs/Point[] points
---
geometry_msgs/Point[] orderedPoints